docs/pages/reference/cli.mdx

```diff
@@ -638,6 +702,40 @@ $ tsh proxy aws --port=10700 --app=awsapp2
 $ aws s3 ls
 ```
 
+### tsh proxy gcloud
+
+Start a local proxy for Google Cloud API access. The user must already be logged
+in to at least one Google Cloud application via Teleport before the proxy can
+start.
+
+```code
+$ tsh proxy gcloud [<flags>] 
+```
+
+#### Flags
+
+| Name | Default Value(s) | Allowed Value(s) | Description |
```

Comment:
This table needs formatting as well, but I wasn't able to find the magic `<br/>` placement to fix it.